src/sentry/utils/performance_issues/detectors/consecutive_http_detector.py

```diff
@@ -80,8 +89,19 @@ def _store_performance_problem(self) -> None:
             cause_span_ids=[],
             parent_span_ids=None,
             offender_span_ids=offender_span_ids,
-            evidence_data={},
-            evidence_display=[],
+            evidence_display=[
+                IssueEvidence(
+                    name="Transaction Name",
+                    value=self._event.get("transaction", ""),
+                    important=True,
+                ),
+            ]
+            + offender_span_evidence_display,
+            evidence_data={
+                "parent_span_ids": [],
```

Comment:
could you also add `op` here because we don't explicitly pass `op` to IssueOccurrence so we need to store it in `evidence_data`

Reply:
done!